app/models/setting/puppet.rb

```diff
@@ -1,7 +1,7 @@
 class Setting::Puppet < Setting
   def self.default_settings
     [
-      self.set('puppet_interval', N_("Puppet interval in minutes"), 30, N_('Puppet interval')),
+      self.set('puppet_interval', N_("Duration in minutes after servers reporting via Puppet are classed as out of sync."), 35, N_('Puppet interval')),
```

Comment:
Is this setting actually used anywhere? I couldn't find anything that uses it, but i might be missing something

Reply:
I was asking the same yesterday: https://github.com/theforeman/foreman/blob/779fea6cb1bbc3468e4f9ebf63cc037a1266e993/app/models/host/managed.rb#L103